packages/contentful--create-contentful-app/src/app-actions/javascript/build-actions.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  try {
+    console.log('Building app actions');
+    mkdirSync('build/actions', { recursive: true });
+
+    const actionsFiles = readdirSync('actions', {
+      encoding: 'utf-8',
+    });
+
+    actionsFiles.forEach((fileName) => {
+      const action = readFileSync(`actions/${fileName}`, {
+        encoding: 'utf-8',
+      }).toString();
+
+      writeFileSync(`build/actions/${fileName}`, action);
+      console.log('App actions successfully built');
+    });
+  } catch (e) {
+    console.log('Failed to build app actions.');
+    console.log(e);
+    return;
+  }
+};
+
+buildActions();
```

Comment:
Should this whole file be one level up folder-wise?

Reply:
No I think it's fine there. It will be at the root level of the example when copied over. But, this is just to keep things organized in CCA itself. That file is only necessary for the js example, so I think it should stay in the js folder.